diff --git a/samples/python/utils/install.py b/samples/python/utils/install.py
index a683813..d84caa1 100644
--- a/samples/python/utils/install.py
+++ b/samples/python/utils/install.py
@@ -39,7 +39,7 @@ ARIA2C_URL = "https://github.com/aria2/aria2/releases/download/release-1.36.0/ar
 def setup_qai_hub(hub_id):
     if os.path.isfile(QAI_HUB_CONFIG):
         shutil.copy(QAI_HUB_CONFIG, QAI_HUB_CONFIG_BACKUP)
-    run_command(f"qai-hub.exe configure --api_token {hub_id} > NUL", False)
+    run_command(f"qai-hub configure --api_token {hub_id} > NUL", False)
 
 
 def reset_qai_hub():
